main.go

```diff
@@ -61,16 +61,22 @@ var cookCmd = &cobra.Command{
 
 var cleanCmd = &cobra.Command{
 	Use:   "clean",
-	Short: "Clean a recipe",
+	Short: "Clean a playground session",
```

Comment:
I am wondering if we should change this from `clean` to `stop`. Clean conveys that after the command everything has been removed and there is nothing else to do. But, you would still have to remove the output folder to really clean the execution.

Reply:
`clean` seems consistent with `cook` :slightly_smiling_face: So maybe we can make two changes at once to keep it consistent:
- `cook` -> `start`
- `clean` -> `stop`

WDYT?